packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('BottomSheetThemeData copyWith, ==, hashCode basics', () {
+    expect(const BottomSheetThemeData(), const BottomSheetThemeData().copyWith());
+    expect(const BottomSheetThemeData().hashCode, const BottomSheetThemeData().copyWith().hashCode);
+  });
+
+  test('BottomSheetThemeData null fields by default', () {
+    const BottomSheetThemeData snackBarTheme = BottomSheetThemeData();
```

Comment:
It seems a little strange to call this variable `snackBarTheme`.

Reply:
Oops, fixed